src/AdminConsole/Components/Pages/App/Settings/AuthenticationConfiguration/AuthenticationConfigurationForm.razor

```diff
@@ -0,0 +1,56 @@
+@using Fido2NetLib.Objects
+@inherits BaseRazorComponent
+
+@inject ILogger<AuthenticationConfigurationForm> Logger
+@inject IScopedPasswordlessClient Passwordless
+@inject ICurrentContext CurrentContext
+
+<EditForm Model="FormModel" OnValidSubmit="SubmitAsync" FormName="@FormName" class="space-y-6">
+    <DataAnnotationsValidator/>
+    <ValidationSummary/>
+    <div class="sm:w-full sm:max-w-md">
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Purpose">Purpose:</label>
+        <InputText @bind-Value="@FormModel!.Purpose" id="Purpose" disabled="@IsPurposeReadOnly" class="text-input"/>
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="UserVerification">User Verification:</label>
+        <InputSelect id="UserVerification" @bind-Value="@FormModel.UserVerificationRequirement" TValue="UserVerificationRequirement" class="form-select">
+            @foreach (var uv in Enum.GetValues<UserVerificationRequirement>())
+            {
+                <option value="@uv">@uv</option>
+            }
+        </InputSelect>
+        <label class="block text-sm font-medium leading-6 text-gray-900 my-2" for="Seconds">Seconds:</label>
+        <InputNumber id="Seconds" @bind-Value="@FormModel.Seconds" min="0" class="text-input"/>
+        <button class="btn-primary mt-3" type="submit">Submit</button>
+    </div>
+</EditForm>
+
+@code {
+    [Parameter] 
+    public EventCallback<AuthenticationConfigurationFormModel> OnSubmitFunction { get; set; }

```

Comment:
You might want to handle specific logic for the form submission in this component. And only chain this at the end of the OnSubmit handler of the form to invoke the callback function if you need to pass back information. But since it's SSR, there's unlikely use for this.

It makes more sense for interactivity but very likely useless here.